tests/foreman/ui/test_contenthost.py

```diff
@@ -1475,11 +1473,9 @@ def test_content_access_after_stopped_foreman(
 
     :CaseComponent: Infrastructure
 
-    :Assignee: lpramuk
-
     :parametrized: yes
     """
-    sat = foreman_service_teardown
+    sat = default_sat
```

Comment:
Can I suggest just using `default_sat` in the test function instead of re-labeling it? With the old fixture name it was cumbersome, but `default_sat` is simple.

Reply:
Hi, looks like I made a commit for that on the 1st of April